WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/Package Creation/ShippingLabelServicePackageList.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+import Yosemite
+
+struct ShippingLabelServicePackageList: View {
+    @Environment(\.presentationMode) var presentation
+    @StateObject var viewModel = ShippingLabelServicePackageListViewModel()
+    let packagesResponse: ShippingLabelPackagesResponse?
+    let safeAreaInsets: EdgeInsets
+
+    var body: some View {
+        LazyVStack(spacing: 0) {
+            ListHeaderView(text: Localization.servicePackageHeader, alignment: .left)
+                .padding(.horizontal, insets: safeAreaInsets)
+
+            /// Packages
+            ///
+            ForEach(viewModel.predefinedOptions, id: \.title) { option in
+
+                ListHeaderView(text: option.title.uppercased(), alignment: .left)
+                    .padding(.horizontal, insets: safeAreaInsets)
+                ForEach(option.predefinedPackages) { package in
+                    let selected = package == viewModel.selectedPackage
+                    SelectableItemRow(title: package.title,
+                                      subtitle: package.dimensions + " \(viewModel.dimensionUnit)",
+                                      selected: selected)
+                        .onTapGesture {
+                            viewModel.selectedPackage = package
+                        }
+                        .padding(.horizontal, insets: safeAreaInsets)
+                        .background(Color(.systemBackground))
+                    Divider().padding(.leading, Constants.horizontalPadding)
```

Comment:
I'd say the divider needs padding for safe area too, since we're currently having different padding for the divider and the tab bar and it doesn't look neat I think:

![Simulator Screen Shot - iPhone 12 Pro - 2021-08-30 at 11 39 50](https://user-images.githubusercontent.com/5533851/131286857-85bc5c7f-32a7-4ef1-8a7c-62ec91983473.png)

Reply:
Yep! Thanks for catching that; fixed in e48ab4a.